_posts/2018-08-14-apu2-validation-with-rte.md

```diff
@@ -0,0 +1,251 @@
+---
+post_title: PC Engines APU2 platform validation with RTE
+author: Artur Raglis
+layout: post
+published: true
+post_date: 2018-08-20 15:00:00
+
+tags:
+        - AMD
+        - APU2
+        - RTE
+categories:
+        - Firmware
+---
+
+## Introduction
+
+Remote work is trending nowadays. The best example is the IT industry - purely
+software tasks with handheld devices allow you to work practically from
+anywhere. This approach saves a big amount of time and makes a job easier.
+Unfortunately, as an embedded / firmware developer, there are often situations
+when interaction with hardware such as a power cycle is required. This leads to
+a barrier for successful remote work. Therefore, there is a possibility of
+connecting the target platform to the Remote Testing Environment product giving
+ready to go full validation set. In this blog post, I will explain the required
+steps to connect our device under test with RTE and then show the possible use
+cases for the whole setup.
```

Comment:
@artur-rs what about link to seles page?

Reply:
@pietrushnic added sales information at the end of the post (with link to RTE website)